packages/application-config/schema.json

```diff
@@ -0,0 +1,133 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://unpkg.com/@commercetools-frontend/application-config/schema.json",
+  "title": "JSON schema for Custom Application configuration files",
+  "type": "object",
+  "definitions": {
+    "envVariablePlaceholder": {
+      "type": "string",
+      "pattern": "(\\$\\{env:\\w+\\})+"
+    },
+    "cspDirective": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "uniqueItems": true
+    }
+  },
+  "properties": {
+    "id": {
+      "description": "The ID of the Custom Application (when you register the application in the Merchant Center)",
+      "type": "string"
+    },
+    "name": {
+      "description": "The name of the Custom Application",
+      "type": "string"
+    },
+    "entryPointUriPath": {
+      "description": "The unique route path of the Custom Application. This is the identifier that the Merchant Center Proxy uses to match the HTTP request and forward the request to the Custom Application. This value also needs to be used in the application client side routes. The value must be the same as the `navbarMenu.uriPath` value.",
+      "type": "string"
+    },
```

Comment:
The `entryPointUriPath` is a "special" property that intends to clear up possible confusions.
The property itself is not used in the config, it just serves an informative purpose and a reminder to the user that this is the entry point of the Custom Application.

Effectively, this value should be the same as the `navbarMenu.uriPath`, which is what the MC proxy uses to match the incoming request.